src/content/updates/release-information/highlights.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Highlights
+tabs: ['Highlights', "Changelog", 'Known Issues']
```

Comment:
```suggestion
tabs: ['Highlights', "Changelog", 'Known issues']
```

minor thing, probably want the second word lowercase on all pages "Known issues"